src/react/PipelineSourcePage.js

```diff
@@ -95,20 +95,32 @@ export default class PipelineSourcePage extends React.Component {
 
     switch (this.state.phase) {
       case "ok":
-        const { name, source, description } = this.state.pipelineSource;
+        const { name, source, schema, description } = this.state.pipelineSource;
 
         innerHTML = (
           <>
           <Row>
             <Col>
               <h5 className="mt-2 mb-2">Name</h5>
               <p>{name}</p>
-              <h5 className="mt-2 mb-2">Description</h5>
-              <p>{description}</p>
+              { description && description !== '' &&
```

Comment:
`description && description !== '' && ...` is the same as `description && ...` so you could just use the latter. The same goes for `schema` below